packages/orchestration/test/examples/sendAnywhere.test.ts

```diff
@@ -164,7 +164,7 @@ test('send using arbitrary chain info', async t => {
       '@type': '/ibc.applications.transfer.v1.MsgTransfer',
       receiver: 'cosmos1destAddr',
       sender: execAddr,
-      sourceChannel: 'channel-1',
+      sourceChannel: 'channel-5',
```

Comment:
:clap: I realized that should be higher the other day but didn't do anything about it.